packages/create-cloudflare/templates/next/wrangler.toml

```diff
@@ -0,0 +1,55 @@
+name = "<TBD>"
+compatibility_date = "<TBD>"
```

Comment:
don't we also need to enable the nodejs_compat flag by default to make ALS work in Next which natively depends on it?

Reply:
yeah, I just left it in the `pages:preview` script, let me move it here, thanks for noticing it 🙂

Reply:
compat flag added 🙂👍